### ISSUE TITLE
Question on usage

### ISSUE THREAD

**Issue by andrepferreira** [IOT-DSA/dslink-dart-rest#5]
Hi there
I am trying to make use of this dslink, but cannot get it to work. I assume this link allows API style access to any of the nodes in a running system. 

I am trying, as a test, to read the Memory Usage value from the System node, but I am not sure how the REST server and the node are connected. Do I first create a server:port, then do I need to connect the node I want to access to this server node on the dataflow canvas? Or by creating the server, do all the nodes become accessible? I have tried both by always get no response from the server:port combination. 

Could you perhaps show a full url for a query of the Memory Usage value so that I can get an idea?

Thanks

**Comment by judders** [comment]
Try watching this Video.
https://www.youtube.com/watch?v=IbHZS5jQLMA
They are using the REST server

**Comment by tishma** [comment]
I agree this is a real issue. The README doesn't make it clear at all. It looks like:
1. Data Client mode REST server can READ single or multiple values from DSA server's data tree. Or it can be used to set values by invoking nodes and passing value to them.
2. Data Host mode REST server hosts a data tree of its own. CRUD operations can be applied to data hosted in such way. Integrations with DSA server's data can only happen via workflows.

A confirmation or clarification in README would be greatly appreciated.

**Comment by tishma** [comment]
@andrepferreira `http://localhost:8020/downstream/System/Memory_Usage` would be the correct url to read memory usage, provided that port `8020` is setup in Data Client mode.
